src/resizable.rs

```diff
@@ -133,6 +181,15 @@ impl<T> Queue<T> {
                     };
                     self.capacity.fetch_sub(1, Ordering::Relaxed);
                 }
+                // Currently this is a no-op because this resize() function
```

Comment:
What do you mean by that? `resize()` takes a `&self` not a `&mut self`. :thinking: